portal/code/company-portal/src/components/cax-companyData.tsx

```diff
@@ -15,15 +15,78 @@
 import * as React from 'react';
 import { observer } from 'mobx-react';
 import { Row } from 'react-bootstrap';
+import { observable } from 'mobx';
+import { getCompanyDetails } from '../helpers/utils';
 import { withTranslation, WithTranslation } from 'react-i18next';
 import { AiOutlineQuestionCircle, AiOutlineCalendar } from 'react-icons/ai'
 import DatePicker from "react-datepicker";
-import SearchInput from 'react-search-input'
+import SearchInput from 'react-search-input';
+import { FetchBusinessPartnerDto } from "../data/companyDetailsById"
+import { toJS } from 'mobx'
+
+interface ISelectableOption<T = any> {
```

Comment:
Could the change from cax-responsibilities regarding these interfaces from the react library also be applied here?

Reply:
@vsamarora will raise another PR to address that.